src/components/portal/portal.tsx

```diff
@@ -77,6 +77,9 @@ export class Portal {
     @Prop()
     public parent: HTMLElement = document.body;
 
+    @Prop()
+    public inheritParentWidth = false;
```

Comment:
Note to self: Since `limel-portal` isn't documented for external use, this isn't a breaking change.

Reply:
And since it has a default value I guess its just a minor change anyway? :)

Reply:
In the technical sense, yes. But I'd still consider it breaking if it would break the consumer's UI. Which, well, it might or it mightn't I guess. Good thing we can ignore that regardless! 😄